Sources/TuistKit/GraphMappers/GraphMapperProvider.swift

```diff
@@ -30,6 +31,8 @@ final class GraphMapperProvider: GraphMapperProviding {
             mappers.append(CacheMapper())
         }
 
+        mappers.append(SigningMapper())
```

Comment:
Since we have now the concept of project mappers, I think we should turn this one into a [ProjectMapper](https://github.com/tuist/tuist/blob/master/Sources/TuistCore/Graph/Mappers/ProjectMapper.swift) because it doesn't need any knowledge on the graph.

Reply:
Good idea, done!